.changeset/popular-cups-help.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/custom-field-view': major
```

Comment:
this may be legit name for future package, should this change to something like `@keystone-next/example-custom-field-view`

Reply:
Yep, good catch 👍

Reply:
```suggestion
'@keystone-next/example-custom-field-view': major
```